### DISCUSSION TITLE
Regarding the issue of screen recording on Android phones and "Could not load rulesets from directory."

### DISCUSSION THREAD

**Discussion by Kinn2021** [ppy/osu#14433] [Q&A]
**Describe the bug:** When I entered osu!lazer on an Android phone and turned on screen recording, the sound ripped and the response slowed. I seem to be able to rule out the low specs on the phone as it works fine with other big games.
In addition, “Could not load rulesets from directory. This error has been automatically reported to the devs.” This problem always comes up every time I go into osu!lazer, and after multiple updates the problem still hasn't been resolved.

**Screenshots or videos showing encountered issue:** [tweet](https://twitter.com/fuka_zora/status/1428880445702172672?s=20)

**osu!lazer version:** 2021.820.0-lazer (Android)

**Logs:**

**Comment by frenzibyte** [comment]
The portion about "could not load rulesets from directory" is a duplicate of https://github.com/ppy/osu/issues/6507, which is not fatal and wouldn't cause any problems. (in fact, you may potentially have it resolved in the next release)

**Comment by frenzibyte** [comment]
Can you state the screen recorder you're using for reference/reproduction at our end. Also I would be interested in hearing whether this issue occurs before 2021.808.0 where the audio subsystem was switched to using BASS mixers (e.g. [2021.731.0](https://osu.ppy.sh/home/changelog/lazer/2021.731.0))

**Reply by Kinn2021** [reply]
> 2021.808.0

I am using a Huawei phone and Android 10,EMUI 10.1.1 comes with a screen recording tool. This problem also happened before 2021.808.0.
